infrastructure/oss/types.go

```diff
@@ -197,8 +236,14 @@ func (i *ossIssue) ToIssueSeverity() snyk.Severity {
 	}
 	return sev
 }
-func (i *ossIssue) AddCodeActions(learnService learn.Service, ep error_reporting.ErrorReporter) (actions []snyk.
+func (i *ossIssue) AddCodeActions(learnService learn.Service, ep error_reporting.ErrorReporter,
+	affectedFilePath string, issueRange snyk.Range) (actions []snyk.
 	CodeAction) {
+	quickFixAction := i.AddQuickFixAction(affectedFilePath, issueRange)
```

Comment:
this should probably be guarded by a check of the package manager.

Reply:
The `AddQuickFixAction` function will already return `nil` if there isn't one, is there something else we'd like to guard by package manager?

Reply:
Sounds good.